### PR TITLE
gitstatus: Add informative warning in case dir is not found

### PR DESCRIPTION
**Pull request by NoahGorny**
I think we should gradually see (and encourage) more PR's like this, which in the long run will ensure there are many logs throughout `bash-it`

### CODE REVIEW

File: plugins/available/gitstatus.plugin.bash [hunk 1]
```diff
@@ -21,4 +21,6 @@ if [[ -d ${SCM_GIT_GITSTATUS_DIR} ]]; then
   # Start the actual gitstatus binary
   gitstatus_stop && gitstatus_start -s -1 -u -1 -c -1 -d -1
   export SCM_GIT_USE_GITSTATUS=true
+else
+	_log_warning "Could not find gitstatus directory in ${SCM_GIT_GITSTATUS_DIR}. Please specify directory location using SCM_GIT_GITSTATUS_DIR."
```
Comment (davidpfarrell) [inline plugins/available/gitstatus.plugin.bash:25]: Thinking these types of messages should also add some type of standard `"You can disable the plugin ..."` messaging and / or "See bashit help ..." messaging

**Comment by davidpfarrell** [comment]
This seems like the type of use-case that the logging was added for, so it's a 👍 from me.

File: plugins/available/gitstatus.plugin.bash (hunk 1 above)
Comment (NoahGorny) [reply plugins/available/gitstatus.plugin.bash:25]: Not sure how can we standardize that. Some logs should display that but some logs should not...
We can add this manually to each message that needs it

**Comment by nwinkler** [comment]
Yes, that's a great use case for the log functionality.

**Pull request merged**